src/Libraries/DynamoWatch3D/Watch3D.cs

```diff
@@ -96,42 +100,41 @@ public void CustomizeView(Watch3D model, NodeView nodeView)
                     nodeView.Dispatcher.Invoke(
                         new Action<object>(RenderData),
                         DispatcherPriority.Render,
-                        obj));
-
-            View.Loaded += View_Loaded;
-           
-        }
-
-        void View_Loaded(object sender, RoutedEventArgs e)
-        {
-            watch3dModel.GetBranchVisualization(null);
+                        obj));       
         }
 
         private void UpdateLatestCameraPosition()
         {
-            if (View == null) return;
+            if (watch3dModel.View == null) return;
 
-            var pos = View.View.Camera.Position;
-            var viewDir = View.View.Camera.LookDirection;
+            var pos = watch3dModel.View.View.Camera.Position;
+            var viewDir = watch3dModel.View.View.Camera.LookDirection;
 
             // Convert Helix3D coordinates from +Y up to +Z up.
             watch3dModel.CameraPosition = new Point3D(pos.X, -pos.Z, pos.Y);
             watch3dModel.LookDirection = new Vector3D(viewDir.X, -viewDir.Z, viewDir.Y);
         }
 
+        private IRenderPackage CreateRenderPackageFromGraphicItem(IGraphicItem gItem)
+        {
+            var factory = new HelixRenderPackageFactory();
+            var renderPackage = factory.CreateRenderPackage();
+            gItem.Tessellate(renderPackage, factory.TessellationParameters);
+            return renderPackage;
```

Comment:
This is part of the callback for the `VMDataBridge`. I am of a mind to remove this because it doesn't do what Stephen originally intended it to do...I don't think.